main.go

```diff
@@ -168,9 +164,9 @@ func main() {
 		// Listen on our channel AND a timeout channel - which ever happens first.
 		select {
 		case <-c1:
-			fmt.Println("Simulation Finished Successfully")
+			log.Info("Simulation Finished Successfully")
 		case <-time.After(time.Duration(parameters.SimTimeoutSeconds) * time.Second):
-			fmt.Println("Simulation Timeout")
+			log.Info("Simulation Timeout")
```

Comment:
If you're logging in fatal why are you logging in info?

Reply:
No idea where that came from lol. Thanks